integration/test_detectionlists.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime
+from datetime import timedelta
+
+
+new_user = "integraiton_" + str(int(datetime.today().timestamp())) + "@test.com"
```

Comment:
`integration`

Reply:
The date here does not need to be localized, therefore it makes more sense to use the `now()` function rather than `today()`

https://stackoverflow.com/questions/32517248/what-is-the-difference-between-python-functions-datetime-now-and-datetime-t

Reply:
For the sake of a random generated ID, it does not to be localized, so you can use the `now()` function over the `today()` function off `datetime`.

For the departure date, it does make sense for it to be localized though

Reply:
changed both and fixed spelling.